service/debugger/debugger.go

```diff
@@ -916,7 +917,44 @@ func (d *Debugger) Goroutines(start, count int) ([]*api.Goroutine, int, error) {
 	if err != nil {
 		return nil, 0, err
 	}
+	if start > 0 || nextg >= 0 {
+		// Ensure that the first batch of goroutines always contains all running
```

Comment:
nit: I think we should reword this comment, as we don't make any explicit guarentee that we are returning *all* running goroutines in the first batch, as that's limited by the count the user specifies. It would suffice, I think, to simply say that we sort and prioritize running goroutines first.